easybuild/easyconfigs/m/matplotlib/matplotlib-3.5.1-foss-2020b.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'PythonBundle'
+
+name = 'matplotlib'
+version = '3.5.1'
+
+homepage = 'https://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('libpng', '1.6.37'),
+    ('freetype', '2.10.3'),
+    ('Tkinter', '%(pyver)s'),
+    ('Pillow', '8.0.1'),
+]
```

Comment:
It needs QHull as a dependency, can you add that (also to the intel versions of the EasyConfig)

Reply:
Added QHull to both now. The update commit message actually is correct actually. I have checked with ldd and it always link agains EB provided Qhull even if System Qhull is available.  Now this PR should be OK.